CHANGELOG.md

```diff
@@ -168,6 +168,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [BUG]: flaky test index/80_geo_point/Single point test([#4860](https://github.com/opensearch-project/OpenSearch/pull/4860))
 - Fix bug in SlicedInputStream with zero length ([#4863](https://github.com/opensearch-project/OpenSearch/pull/4863))
 - Fix a bug on handling an invalid array value for point type field #4900([#4900](https://github.com/opensearch-project/OpenSearch/pull/4900))
+- Fix bug of replica shard's translog not purging on index flush.([4928](https://github.com/opensearch-project/OpenSearch/pull/4928))
```

Comment:
please add context that this only happened with segment replication enabled.

Reply:
remove the period in the sentence too

fix the line above as well, remove the extra #4900 while you're at it?